scripts/levelManagement.js

```diff
@@ -0,0 +1,136 @@
+function configureDownloadLevelButton() {
+    var downloadLevelButton = document.getElementById(
+        "selected-level-download"
+    );
+    refreshElement(downloadLevelButton);
+    var downloadLevelButton = document.getElementById(
+        "selected-level-download"
+    );
```

Comment:
That's because the I wasn't sure if refreshElement destroyed the old element or not. I've had problems with similar functions in the past, so I did that just to be safe.